<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookEntryScreen</class>
 <widget class="QMainWindow" name="BookEntryScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1454</width>
    <height>902</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>511</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Book Entry Portal</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toggleButton">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>10</y>
      <width>25</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ðŸ—–</string>
    </property>
   </widget>
   <widget class="QLabel" name="designImage">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>301</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>512</width>
      <height>625</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="bookNameLabel">
       <property name="text">
        <string>Book Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bookNameEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="authorLabel">
       <property name="text">
        <string>Author:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="authorEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="isbnLabel">
       <property name="text">
        <string>ISBN No:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="isbnEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="genreLabel">
       <property name="text">
        <string>Genre:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="csButton">
         <property name="text">
          <string>Computer Science</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="fictionButton">
         <property name="text">
          <string>Fiction</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QRadioButton" name="mathButton">
         <property name="text">
          <string>Mathematics</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="engineeringButton">
         <property name="text">
          <string>Engineering</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="physicsButton">
         <property name="text">
          <string>Physics</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="scifiButton">
         <property name="text">
          <string>Sci-Fi</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QRadioButton" name="nonFictionButton">
         <property name="text">
          <string>non-fiction</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="chemistryButton">
         <property name="text">
          <string>Chemistry</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="selfHelpButton">
         <property name="text">
          <string>Self-Help</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="novelButton">
         <property name="text">
          <string>Novel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QRadioButton" name="historyButton">
         <property name="text">
          <string>History</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QRadioButton" name="literatureButton">
         <property name="text">
          <string>Literature</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="publisherLabel">
       <property name="text">
        <string>Publisher:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="publisherEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="yearLabel">
       <property name="text">
        <string>Published Year:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yearEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="copiesLabel">
       <property name="text">
        <string>Number of Copies:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="copiesEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="uploadLabel">
       <property name="text">
        <string>Upload Image:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uploadButton">
       <property name="text">
        <string>Browse....</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
